### ISSUE TITLE
[Feature Request] underline in user defined colors

### ISSUE THREAD

**Issue by Tanjamuse** [xhuajin/obsidian-tabs#42]
Hi

Now that you've made it possible to decorate the tab-titles, it would be amazing with an option to make the underline the same color as the decoration, if it's possible?

Thank you in advance.

**Comment by xhuajin** [comment]
Did you add the underline with <u>title</u>? My underline color is the same as the title.

**Comment by Tanjamuse** [comment]
No, I mean the underline that signals which tab is active, like in the screenshot below.

![tabs](https://github.com/user-attachments/assets/dfb73340-ed14-4dd5-98a6-d24d2971da81)

**Comment by xhuajin** [comment]
The color of the header and underline is the same without any other css influence. You can check if any css or themes affect the colors.

![image](https://github.com/user-attachments/assets/57c515ca-3b8b-4d49-9c10-8af16ad36c19)

**Comment by Tanjamuse** [comment]
Yes, but with the added option of text-decoration by adding html code like the following: 

`tab: <span style="font: 40px; color:#D05D97">Tab 1</span>`

I would love an option for the underline to be in the same color as the decoration specific for that tab, if that is possible. Alternately a cssclass option for having multiple colors of underlines instead of just the one.

**Comment by xhuajin** [comment]
I see what you mean. That's good advice. I'll optimize it in next version. Thank you for your advice!
